lib/login/ui/sign_in.dart

```diff
@@ -17,113 +20,129 @@ class SignIn extends HookConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final authNotifier = ref.watch(authTokenProvider.notifier);
     final pathForwarding = ref.read(pathForwardingProvider);
+    final controller = ref.watch(backgroundAnimationProvider);
+
     return LoginTemplate(
+      callback: (AnimationController controller) {
+        if (controller.isCompleted) {
+          controller.reverse();
+        }
+      },
       child: AutofillGroup(
-          child: Form(
-        autovalidateMode: AutovalidateMode.onUserInteraction,
-        child: Padding(
-          padding: const EdgeInsets.all(20.0),
-          child: Column(
-            children: [
-              Expanded(
-                flex: 3,
-                child: Align(
-                  alignment: Alignment.centerLeft,
-                  child: Text("MyECL",
-                      style: GoogleFonts.elMessiri(
-                          textStyle: const TextStyle(
-                              fontSize: 40,
-                              fontWeight: FontWeight.bold,
-                              color: Colors.white))),
+        child: Form(
+          autovalidateMode: AutovalidateMode.onUserInteraction,
+          child: Padding(
+            padding: const EdgeInsets.all(20.0),
+            child: Column(
+              children: [
+                Expanded(
+                  flex: 3,
+                  child: Align(
+                    alignment: Alignment.centerLeft,
+                    child: Text("MyECL",
```

Comment:
Use text constant to match your own requirement please

Reply:
Already done in the Refactoring PR : https://github.com/aeecleclair/Titan/pull/91 😉